Makefile

```diff
@@ -94,13 +100,33 @@ unit-test: build-dirs
 	./hack/unit-test.sh
 .PHONY: unit-test
 
+integration-test:
+	@go build ./test
```

Comment:
s/ `@go build ./test`/ `@go test ./test` ? Maybe it's the cause of the failure of the integration test.

Reply:
Thanks for this help!